monitor/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -61,14 +61,14 @@
 				<i class="fa fa-clock-o" aria-hidden="true"></i>
 		</c:otherwise>
 	</c:choose>
-	<strong>${app.name}</strong> (${app.healthUrl})
+	<strong id="name_${app.id}">${app.name}</strong> <span id="url_${app.id}">(${app.healthUrl})</span>
```

Comment:
I would put the `span` inside the parentheses, so that you just need to replace the url.